src/app/public/modules/select-field/select-field.component.ts

```diff
@@ -196,18 +203,21 @@ export class SkySelectFieldComponent implements ControlValueAccessor {
     }
   }
 
+  public onTouched(): void {
+    this._onTouched();
+    this.touched.emit();
```

Comment:
Angular's prescribed way to watch for the `touched` status is through the control's status observable.
```
control.statusChanges.subscribe((status) => {
  // If status is 'touched', do something...
});
```

It feels a little odd to force this control to emit something that's already emitted by the underlying form control?

Reply:
@blackbaud-conorwright that is a good question. `NgModel` also has this property if used in a template driven form.

Reply:
status does not fire on changes to `touched`, we chatted a bit and I've changed this back to a `blur` event, but also updated it to actually only fire when focus leaves the component's bounds (includes modal)